### PR TITLE
Keptn 0.6.1 Rewrite

### PR DESCRIPTION
**Pull request by ghost**
Rewrote this integration to bring it up to date for Keptn 0.6.1

### CODE REVIEW

**Comment by akirasoft1** [comment]
@Dynatrace-Adam-Gardner can you update the codeowners file to remove me and add yourself?

**Comment by ghost** [comment]
Already done: https://github.com/keptn-sandbox/jira-service/pull/4/commits/9c06ebcea510e853ad32db0cb7dd6785bf980424?file-filters%5B%5D=No+extension#diff-5daf978e60e03b3975b485d5f1b449dc

**Comment by ghost** [comment]
Not sure what you mean Jurgen. Yes, I've rewritten this service to be compatible for 0.6.1 and I plan on maintaining this going forward.

**Comment by jetzlstorfer** [comment]
Sorry, didn't see that this line has been removed and thought it is still active.

**Comment by jetzlstorfer** [comment]
Could you also please add a travis file so we can automated builds for this service? 
You can take a look at an example here and use it as a template: https://github.com/keptn-sandbox/slackbot-service/blob/master/.travis.yml

**Comment by robertjahn** [comment]
Update the readme so that you just have to run this rather than clone the repo to install

```
kubectl apply -f https://raw.githubusercontent.com/keptn-sandbox/jira-service/master/jira-service.yaml

kubectl apply -f https://raw.githubusercontent.com/keptn-sandbox/jira-service/master/jira-distributor.yaml
```

**Comment by robertjahn** [comment]
adjust these parameter names in the secret creation to better align to the JIRA naming

```
kubectl -n keptn create secret generic jira-details --from-literal="jira-hostname=***" --from-literal="jira-username=***" --from-literal="jira-token=***" --from-literal="jira-project=***"

jira-hostname to jira-base-url
jira-username to jira-organization
jira-project to jira-project-key
```

**Comment by robertjahn** [comment]
I would also rename the secret from jira-details to jira-service

**Comment by jetzlstorfer** [comment]
I'm fine with accepting this PR and giving ownership to @Dynatrace-Adam-Gardner 

is it ready for merging?

**Comment by ghost** [comment]
@robertjahn have you got this working with the latest commits?
This service should now: 
1. Listen to both `problem.open` and `evaluation-done` events.
2. Create JIRA tickets for both of the above.
3. Flags are set as part of the `jira-service` secret that means you can disable ticket creation for either / both of the above problems.

If Rob gives the go-ahead, I'm happy to merge and take ownership.

**Comment by ghost** [comment]
I've spoken to @robertjahn and he confirms that this version is working:
- Includes the travis file as requested
- Compatible with cloud JIRA for both "classic" and "NextGen" projects
- Creates JIRA tickets for problems (flag available to disable behaviour)
- Creates JIRA tickets for evaluations (flag available to disable behaviour)
- Includes link to Dynatrace problem page (when DT is used) (this enables the TEMPEST JIRA plugin to work)

As such, this PR is ready to merge.

File: jira-service.yaml [hunk 1]
```diff
@@ -15,33 +16,52 @@ spec:
     spec:
       containers:
       - name: jira-service
-        image: mvilliger/jira-service:0.2
+        image: adamgardnerdt/keptn-jira-service:v1
```
Comment (jetzlstorfer) [inline jira-service.yaml]: please change to `keptnsandbox/jira-service:0.1`

Comment (jetzlstorfer) [reply jira-service.yaml]: or :latest is also fine

**Review by jetzlstorfer — CHANGES_REQUESTED** [review]
looks good in general. only one comment to resolve

File: jira-service.yaml (hunk 1 above)
Comment (ghost) [reply jira-service.yaml]: done in 1838381

**Review by jetzlstorfer — APPROVED** [review]
lgtm

**Pull request merged**